server/game_thread.cpp

```diff
@@ -163,8 +163,6 @@ flatbuffers::Offset<FlatBuffGenerated::Mob> createFBMob(flatbuffers::FlatBufferB
 {
 	auto distance = b2Distance(m->body->GetPosition(), player.body->GetPosition());
```

Comment:
Compiler yells that this variable is no longer necessary. If that's true, please remove :)